www.chuyu.me/es.xml

```diff
@@ -4,36 +4,36 @@
   <String Src="%SxSClean%" Dst="Limpieza de WinSxS"/>
   <String Src="%SxSCleanDes%" Dst="No debes preocuparte del espacio libre del disco durante la instalación de las actualizaciones."/>
   <String Src="%CompactOS%" Dst="SO compacto"/>
-  <String Src="%CompactOSDes%" Dst="Al menos hay una reducción del 33% del espacio utilizado, sin simplificar la imagen del SO."/>
+  <String Src="%CompactOSDes%" Dst="Hay una reducción del 33% del espacio utilizado, sin racionalizar la imagen del SO."/>
   <String Src="%EsdSupport%" Dst="Soporte ESD"/>
   <String Src="%EsdSupportDes%" Dst="La conversión de ESD a ISO nunca ha sido tan fácil como ahora."/>
   <String Src="%WindowsUpdate%" Dst="Windows Update"/>
   <String Src="%WindowsUpdateDes%" Dst="Elige tus actualizaciones - y no te preocupes."/>
   <String Src="%SystemRecover%" Dst="Copia de seguridad y recuperación"/>
-  <String Src="%SystemRecoverDes%" Dst="Crear una copia de seguridad o restaurar el sistema sin usar Windows PE."/>
+  <String Src="%SystemRecoverDes%" Dst="Crea una copia de seguridad o restaura el sistema sin usar Windows PE."/>
   <String Src="%Green%" Dst="Portable"/>
   <String Src="%GreenDes%" Dst="GRATUITO, SIN ANUNCIOS y SPYWARE."/>
   <String Src="%AboutChuyu%" Dst="Acerca del equipo de Chuyu"/>
-  <String Src="%AboutChuyuDes%" Dst="Somos perfeccionistas y bastante obsesivos al crear una herramienta que sea ligera, fácil de usar, directa, y sin molestias. Todavía no lo hemos logrado, pero no nos damos por vencidos mientras realizamos un sueño con nuestras propias manos haciendo Dism++: una pequeña herramienta sin fines de lucro y construida con pura persistencia y entusiasmo."/>
-  <String Src="%QQ%" Dst="Grupo oficial"/>
+  <String Src="%AboutChuyuDes%" Dst="Somos perfeccionistas y bastante obsesivos al crear una herramienta que sea ligera, fácil de usar, directa y sin molestias. Todavía no lo hemos logrado, pero no nos damos por vencidos mientras realizamos un sueño con nuestras propias manos haciendo Dism++: una pequeña herramienta sin fines de lucro y construida con pura persistencia y entusiasmo."/>
+  <String Src="%QQ%" Dst="Grupos oficiales"/>
   <String Src="%E-Mail%" Dst="Correo electrónico"/>
   <String Src="%Links%" Dst="Enlaces"/>
   <String Src="%WinOS%" Dst="WINOS Lite System"/>
   <String Src="%ChuyuTream%" Dst="&amp;nbsp;2017&amp;nbsp;Equipo de Chuyu"/>
   <String Src="%ICP%" Dst=""/>
   <String Src="%Document%" Dst="Documentación de Dism++"/>
-  <String Src="%logs%" Dst="Cambios"/>
-  <String Src="%wsuscan2%" Dst="&quot;wsusscn2.cab&quot; es la base de datos para Windows Update. Puede usarlo escribiendo la ruta &quot;Config\wsusscn2.cab&quot;."/>
-</Strings>
\ No newline at end of file
+  <String Src="%logs%" Dst="Registro de cambios"/>
+  <String Src="%wsuscan2%" Dst="&quot;wsusscn2.cab&quot; es la base de datos de Dism++ para Windows Update. Puede usarlo al copiar o sobreescribir en la ruta &quot;Config\wsusscn2.cab&quot;."/>

```

Comment:
Corrections in expressions, and increases for the better understanding of people.